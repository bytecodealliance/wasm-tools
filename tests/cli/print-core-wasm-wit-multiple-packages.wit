// RUN: component embed --dummy --world=bar:bar/my-world % | component wit

package foo:foo {
  interface my-interface {
    foo: func();
  }

  world my-world {
    import my-interface;
  }
}

package bar:bar {
  interface my-interface {
    foo: func();
  }

  world my-world {
    import my-interface;
  }
}
