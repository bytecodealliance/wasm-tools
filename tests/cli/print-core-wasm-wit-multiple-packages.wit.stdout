package root:root;

world root {
  import bar:bar/my-interface;
}
